packages/editor/src/components/ComponentForm/JsonSchemaForm/TopLevelField.tsx

```diff
@@ -32,14 +32,13 @@ const TopLevelField: React.FC<FieldProps> = props => {
     const properties = (schema.properties || {}) as Property;
     Object.keys(properties).forEach(name => {
       const schema = properties[name];
-      if (!schema.title) {
-        schema.name = name;
-      }
+      schema.name = name;
     });
     const grouped = groupBy(properties, c => c.category || 'Basic');
+
     return Object.keys(grouped).map(name => ({
       name,
-      schemas: sortBy(grouped[name], c => c.weight || -Infinity),
+      schemas: sortBy(grouped[name],'weight'),
```

Comment:
what's the difference between the new way and the previous one?

Reply:
think 0 is more reasonable as the highest weighting

Reply:
I use smaller numbers as higher weights,and explicit fallback  is better,I've changed it to `c => c.weight ?? Infinity`